include/session/config/groups/members.hpp

```diff
@@ -489,6 +520,33 @@ class Members : public ConfigBase {
             return copy;
         }
     };
+
+    protected:
+        oxenc::bt_dict extra_data() const override {
+            oxenc::bt_dict result;
+            if (pending_send_ids && !pending_send_ids->empty()) {
+                oxenc::bt_list ids = {};
+                for (const auto& sid : *pending_send_ids)
+                    ids.emplace_back(sid);
+
+                result["pending_send_ids"] = ids;
+            }
+            
+            return result;
+        }
+        
+        void load_extra_data([[maybe_unused]] oxenc::bt_dict extra) override {
+            if (auto it = extra.find("pending_send_ids"); it != extra.end()) {
+                if (std::holds_alternative<oxenc::bt_list>(it->second)) {
+                    auto invitations = std::get<oxenc::bt_list>(it->second);
+                    pending_send_ids = {};
+
+                    for (const auto& sid : invitations)
+                        if (std::holds_alternative<std::string>(sid))
+                            pending_send_ids->emplace(std::get<std::string>(sid));
+                }
+            }
+        }
```

Comment:
And this one can become:
```suggestion
        void load_extra_data(oxenc::bt_dict_consumer&& extra) override {
            if (extra.skip_until("pending_send_ids"))
                extra.consume(pending_send_ids);
        }
```